src/pageLayout/components/UserWidget.tsx

```diff
@@ -78,6 +79,16 @@ const UserWidget: FC<Props> = ({
             />
           )}
         />
+        <FeatureFlag name="unity">
+          <TreeNav.UserItem
+            id="unity-users"
+            label="Unity Users"
+            testID="user-nav-item-unity-users"
+            linkElement={className => (
+              <a className={className} href={`/orgs/${org.id}/unity-users`} />
```

Comment:
Do we want `unity` in our route name and on the Nav Item? Or is that temporary because we have both `users` and `unity users` right now?